docker/README.md

```diff
@@ -0,0 +1,32 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+# What is this?
+It is a docker image for building yosegi.
+
+# Create a docker image
+
+Please execute the command in this directory.
+
+```
+docker build -t yahoojp/yosegi_build_image .
```

Comment:
This script does not work for certification reason.

Sending build context to Docker daemon  6.656kB
Step 1/19 : FROM centos:centos7 as java-base
Get https://registry-1.docker.io/v2/library/centos/manifests/centos7: unauthorized: incorrect username or password

Reply:
Thank you. I forgot login to Docker.